include/stumpless/facility.h

```diff
@@ -389,6 +389,9 @@ stumpless_get_facility_string( enum stumpless_facility facility );
  * @since release v2.1.0.
  *
  * @param facility_string The facility name to get the enum from.
+ * 
+ * @error Clears previous error on success.
+ * @error Raises STUMPLESS_INVALID_FACILITY on invalid input.
```

Comment:
As far as I can tell from the [Doxygen documentation](https://www.doxygen.nl/manual/commands.html) `@error` is not a supported special command. Instead, you can just put these two descriptions into their own paragraph to describe the error behaviors.